Source/Shaders/Model/LegacyInstancingStageVS.glsl

```diff
@@ -4,13 +4,16 @@ void legacyInstancingStage(
     out mat3 instanceModelViewInverseTranspose)
 {
     vec3 positionMC = attributes.positionMC;
-    mat4 instancingTransform = getInstancingTransform();
+    vec3 normalMC = attributes.normalMC;
 
+    mat4 instancingTransform = getInstancingTransform();
+ 
     mat4 instanceModel = instancingTransform * u_instance_nodeTransform;
     instanceModelView = u_instance_modifiedModelView;
     instanceModelViewInverseTranspose = mat3(u_instance_modifiedModelView * instanceModel);
 
     attributes.positionMC = (instanceModel * vec4(positionMC, 1.0)).xyz;
+    attributes.normalMC = (instanceModel * vec4(normalMC, 0.0)).xyz;
```

Comment:
Technically we should be computing the inverse transpose. However, the old `ModelInstanceCollection` did it this way, so I just opened https://github.com/CesiumGS/cesium/issues/10693 instead.